syncer/checkpoint.go

```diff
@@ -244,7 +247,7 @@ func NewRemoteCheckPoint(tctx *tcontext.Context, cfg *config.SubTaskConfig, id s
 		tableName:   dbutil.TableName(cfg.MetaSchema, cfg.Name+"_syncer_checkpoint"),
 		id:          id,
 		points:      make(map[string]map[string]*binlogPoint),
-		globalPoint: newBinlogPoint(minCheckpoint, nil, minCheckpoint, nil),
+		globalPoint: newBinlogPoint(binlog.NewLocation(cfg.Flavor), binlog.NewLocation(cfg.Flavor), nil, nil),
```

Comment:
Why do we use the same function twice here? Can we combine them to one?

Reply:
update in if a59f087

Reply:
Is it fine for these two locations ref the same `.GTIDSet`?

Reply:
I revert this change in 9f48852